src/NzbDrone.Core/Download/Clients/DownloadStation/Proxies/DownloadStationTaskProxyV2.cs

```diff
@@ -32,6 +35,21 @@ public void AddTaskFromData(byte[] data, string filename, string downloadDirecto
             {
                 requestBuilder.AddFormParameter("destination", $"\"{downloadDirectory}\"");
             }
+            else
+            {
+                _logger.Trace("Empty destination folder in setting, getting default destination folder from DownloadStation.");
```

Comment:
I think this would be better phrased as "No directory configured in settings; falling back to client default destination folder

Reply:
I made the change